src/dotnet/APIView/APIViewWeb/Managers/ReviewManager.cs

```diff
@@ -352,23 +352,23 @@ public async Task UpdateReviewBackground(HashSet<string> updateDisabledLanguages
             {
                 if (updateDisabledLanguages.Contains(language))
                 {
-                    _telemetryClient.TrackTrace("Background task to update API review at startup is disabled for langauge " + language);
+                    _telemetryClient.TrackTrace("Background task to update API review at startup is disabled for language " + language);
                     continue;
                 }
 
                 var languageService = GetLanguageService(language);
                 if (languageService == null)
                     return;
 
-                // If review is updated using devops pipeline then batch process update review requests
+                // If review is updated using Azure DevOps pipeline then batch process update review requests
                 if (languageService.IsReviewGenByPipeline)
                 {
                     await UpdateReviewsUsingPipeline(language, languageService, backgroundBatchProcessCount);
                 }
                 else
                 {
                     var reviews = await _reviewsRepository.GetReviewsAsync(false, language, fetchAllPages: true);
-                    foreach (var review in reviews.Where(r => IsUpdateAvailable(r)))
+                    foreach (var review in reviews.Where(r => IsUpdateAvailable(r) || r.Revisions.Any(rev => string.IsNullOrEmpty(rev.SingleFile?.PackageVersion))))
```

Comment:
we want to update the review if we got the package version, right?

Reply:
This check is to find the eligible reviews to rebuild. We will rebuild review data if either parser version is different or if package version is missing in review metadata.